src/main/scala/devices/spi/OM_SPI.scala

```diff
@@ -4,7 +4,16 @@ package sifive.blocks.devices.spi
 import freechips.rocketchip.diplomaticobjectmodel.model.{OMDevice, OMInterrupt, OMMemoryRegion}
 
 case class OMSPI(
-  numCS: Int,
+  rxDepth: Int,
+  txDepth:Int,
```

Comment:
can you add a space here (manual lint ;)

Reply:
Sure!